src/components/SelectContractFilter.vue

```diff
@@ -13,11 +13,14 @@
     @input="changeContract"
   >
     <template #selection="contract">
-      {{
-        disabled
-          ? "Du hast noch keinen Vertrag."
-          : contract.item.name + contractStatus(contract.item)
-      }}
+      <div v-if="disabled">
+        {{ "Du hast noch keinen Vertrag. " }}
+        <router-link v-if="disabled" to="/contracts">Hier</router-link>
```

Comment:
Ich weiß nicht, ob wir das zum Problem machen - aber aus Sicht der Barrierearmut wird das Verlinken von "Hier" immer kritisiert weil das für Screenreader nicht transparent ist. Ist aber vermutlich sowieso fraglich, wie gut Clock mit Screenreader funktioniert... Dennoch sollte der Link den ganzen Satz "Hier kannst Du..." umfassen, dann ist es gut.

Mit locale und Links gibt es eine Präzedenz in `DashboardWelcome`.

Eine andere Idee, die ich hatte: Das Dropdown könnte reaktiv sein und beim Aufklappen wie folgt aussehen:
Erstes Item "Du hast noch keinen Vertrag"
Zweites Item "+ Einen neuen Vertrag anlegen"
Das fände ich ziemlich schick, aber ich weiß nicht, ob die Komponente so aufgebohrt werden kann. Die Lösung mit Link ist für den ersten Anlauf gut!

Reply:
und vllt. schöneres Wording: "Hier kannst du einen neuen Vertrag anlegen."